pkg/controller/controller_binding.go

```diff
@@ -166,7 +166,7 @@ func (c *controller) reconcileBinding(binding *v1alpha1.Binding) error {
 
 		var parameters map[string]interface{}
 		if binding.Spec.Parameters != nil {
-			parameters, err = unmarshalParameters(binding.Spec.Parameters.Raw)
+			parameters, err = unmarshalRawParameters(binding.Spec.Parameters.Raw)
```

Comment:
@duglin I'm planning to do this as a separate PR once we merge this one. One step at a time :)